pom.xml

```diff
@@ -232,13 +232,13 @@
     <dependency>
       <groupId>org.hibernate</groupId>
       <artifactId>hibernate-validator</artifactId>
-      <version>5.4.3.Final</version>
+      <version>6.2.5.Final</version>
```

Comment:
Same for these 2: groupId should be `org.hibernate.validator`

Reply:
Updated